index.html

```diff
@@ -7,7 +7,20 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="container"></div>
+    <div class="container">
+      
+      <h2>대기중 (<span id="count-waiting"></span>)</h2>
+      <ul class="list waiting"></ul>
+
+      <h2>완료됨 (<span id="count-complete"></span>)</h2>
+      <ul class="list complete"></ul>
+
+      <form class="input">
+        <input id="todo-new" type="text" placeholder="새로운 할일 추가">
```

Comment:
이부분도 마찬가지로 통일해주시면 좋을 것 같습니다~

```suggestion
        <input id="todo-input" type="text" placeholder="새로운 할일 추가">
```



추가적으로, id와 class를 섞어 사용하여주셨는데 class만 가지고도 구성할 수 있는 네이밍 규칙이 있답니당. 
편하신대로 사용하시면 좋을 것 같아요~[참고](http://getbem.com/introduction/)

Reply:
id와 class 중 어떤걸 사용할지 고민 될 때가 있는데, 정하는 기준이 있을까요 ??

Reply:
네 ! 안 그래도 섞어 써서 정신 없었는데 링크 읽어볼게요 ㅎㅎ